src/queries.js

```diff
@@ -202,6 +266,69 @@ export const ContenfulPage = graphql`
         contentful_id
         embedHtml
       }
+      ... on ContentfulBlogPost {
+        internal {
+          type
+        }
+        blogPostTitle
+        blogPostLanguage
+        slug
+        blogPostDate
+        blogPostDescription
+        coverImage {
+          file {
+            url
+            fileName
+          }
+          title
+        }
+        blogPostContent {
```

Comment:
Technically we do not need to query the blogPostContent to be able to embed the blogpost cards on the pages. But I think it is reasonable also to keep these types the same to make them easy to modify as necessary, so no need to change this. I probably shouldn't even comment about this then 😄